lyo-sandbox/sandbox-twin-webapp/src/main/java/se/ericsson/cf/scott/sandbox/TwinManager.java

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2012 IBM Corporation and others.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ *  
+ *  The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ *  and the Eclipse Distribution License is available at
+ *  http://www.eclipse.org/org/documents/edl-v10.php.
+ *  
+ *  Contributors:
+ *  
+ *	   Sam Padgett	       - initial API and implementation
+ *     Michael Fiedler     - adapted for OSLC4J
+ *     Jad El-khoury        - initial implementation of code generator (https://bugs.eclipse.org/bugs/show_bug.cgi?id=422448)
+ *     Matthieu Helleboid   - Support for multiple Service Providers.
+ *     Anass Radouani       - Support for multiple Service Providers.
+ *
+ * This file is generated by org.eclipse.lyo.oslc4j.codegenerator
+ *******************************************************************************/
+
+package se.ericsson.cf.scott.sandbox;
```

Comment:
I know, it may be too late, but, if we want to have correct reversed domain name, it should be "net.ericsson.cf.scott..."

Reply:
We can even change it to `eu.scott.warehouse`